From 92db841a0d8449b46ce9d9e76bb4a23e5b370f9c Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Fri, 11 Sep 2020 07:07:00 +0000
Subject: [PATCH] GCC: remove noexcept from PDF Thumbnail

Move constructor/operator can't be noexcept, because of SkBitmap
member has no noexcept move constructor/operator.
---
 pdf/thumbnail.cc | 4 ++--
 pdf/thumbnail.h  | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/pdf/thumbnail.cc b/pdf/thumbnail.cc
index d46b20b..82f4790 100644
--- a/pdf/thumbnail.cc
+++ b/pdf/thumbnail.cc
@@ -113,9 +113,9 @@ Thumbnail::Thumbnail(const gfx::Size& page_size, float device_pixel_ratio) {
   DCHECK(success);
 }
 
-Thumbnail::Thumbnail(Thumbnail&& other) noexcept = default;
+Thumbnail::Thumbnail(Thumbnail&& other) = default;
 
-Thumbnail& Thumbnail::operator=(Thumbnail&& other) noexcept = default;
+Thumbnail& Thumbnail::operator=(Thumbnail&& other) = default;
 
 Thumbnail::~Thumbnail() = default;
 
diff --git a/pdf/thumbnail.h b/pdf/thumbnail.h
index 4b2bdef..d62a56a 100644
--- a/pdf/thumbnail.h
+++ b/pdf/thumbnail.h
@@ -17,8 +17,8 @@ class Thumbnail final {
  public:
   Thumbnail();
   Thumbnail(const gfx::Size& page_size, float device_pixel_ratio);
-  Thumbnail(Thumbnail&& other) noexcept;
-  Thumbnail& operator=(Thumbnail&& other) noexcept;
+  Thumbnail(Thumbnail&& other);
+  Thumbnail& operator=(Thumbnail&& other);
   ~Thumbnail();
 
   SkBitmap& bitmap() { return bitmap_; }
-- 
2.26.2

